### DISCUSSION TITLE
Optimized the display effect of inlays

### DISCUSSION THREAD

**Discussion by ofzo** [helix-editor/helix#9699] [Ideas]
![image](https://github.com/helix-editor/helix/assets/18000037/95ab772b-63fd-478f-ad97-daca4ff783bb)

when a selection contain inlay text, inlay text should be rendered by the selection's background

**Comment by pascalkuthe** [comment]
behaviour is intentional and consistent with how vs code renders inlay hints

**Reply by ofzo** [reply]
<img width="753" alt="image" src="https://github.com/helix-editor/helix/assets/18000037/c3fa1c5a-a30d-4c7a-a875-4fec5e9c8364">

this code render in vscode, the inlay text's background is equivalent to the selection's background
